packages/files-ui/cypress/tests/file-management-spec.ts

```diff
@@ -502,5 +504,41 @@ describe("File management", () => {
       fileInfoModal.closeButton().click()
       fileInfoModal.body().should("not.exist")
     })
+
+    it("can download a file from file browser", () => {
+      const fileName = "text-file.txt"
+      const downloadsFolder = Cypress.config("downloadsFolder")
```

Comment:
I would consider storing the relative path as well so that it can be reused
```suggestion
      const downloadsFolder = Cypress.config("downloadsFolder")
      const fileRelativePath = `../fixtures/uploadedFiles/${fileName}`
```